ui/src/components/directory/landingPage/PieChartDashCard.vue

```diff
@@ -44,7 +46,7 @@ function setChartSize(): void {
   const reportTable = document.getElementsByClassName("report-table")[0] as HTMLElement;
   const container = document.getElementById(props.id) as HTMLElement;
   if (!container) {
-    LoggerService.error(undefined, `Failed to set chart size for element id: ${props.id}`);
+    log.error(() => `Failed to set chart size for element id: ${props.id}`);
     return;
```

Comment:
(not part of this pr changes) but this is bad practice - should refactor to if/else instead to avoid executing code that is not relevant or explicitly returning false/undefined if the return value is used